python/examples/llama-index/backend/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use an official Python runtime as a parent image
+FROM python:3.11-slim as builder
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends wget \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+# Install poetry
+RUN pip install poetry
```

Comment:
Optional: The final production image does not actually need poetry, which is used to install dependencies. If you want to optimize the production image, you can use poetry to install the dependencies in a `deps` or `builder` stage, then copy the dependencies over to the production image. (Not required, just a cool thing you can do)

Reply:
This is an awesome suggestion, but I'm going to move on for now, thanks!